components/steering_wheel/HW/HW_intc.c

```diff
@@ -12,7 +12,6 @@
 #include "stm32f1xx.h"
 #include "HW_intc.h"
 
-

```

Comment:
please just completely revert the changes to this file, it shouldn't even be included in the pull request.